doc/user/interop.md

```diff
@@ -310,6 +310,8 @@ an integer, or anything else
 
 `object.to_a` and `object.to_ary` calls `Truffle::Interop.to_array(object)`
 
+`object.equal?(other)` returns whether object is the same as other, like BasicObject#equal?
```

Comment:
Can you use code quotes around `BasicObject#equal?`, and clarify 'same' - is this reference equality?

Reply:
Done.